### PR TITLE
[News] 2020-06-08 PMD CPD support

### PR DESCRIPTION
**Pull request by ybiquitous**
Related to #431

### CODE REVIEW

File: docs/news/2020.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ This document describes notable changes on Sider in the year 2020.
 
 ---
 
+## New PMD CPD support
+
+<time class="news-date" datetime="2020-06-08">June 8, 2020</time>
+
+We are happy to announce that [PMD CPD](../tools/others/pmd-cpd.md) is newly supported! (yet beta) 🎉
+
+This tool is a "Copy/Paste Detector" to find duplicated code in multiple languages like Java, Ruby, Python, and any more.
```
Comment (ybiquitous) [inline docs/news/2020.md:20]: > __[LanguageTool]__ Did you mean the adverb "anymore"? ([view](https://sider.review/gh/repos/142577083/pulls/437#issue-3934599))
>
> <details>
>
> | Rule | Type | Category | Replacements |
> | ---- | ---- | -------- | ------------ |
> | `ANY_MORE` | misspelling | US_SPECIFIC_TYPOS | `anymore` |
>
> You can [close this issue](https://sider.review/gh/repos/142577083/pulls/437#issue-3934599) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

No, "any more" is correct here.

Comment (yykamei) [reply docs/news/2020.md:20]: OK, you're right. By the way, I think "and so on" could be also used here.

Comment (ybiquitous) [reply docs/news/2020.md:20]:
```suggestion
This tool is a "Copy/Paste Detector" to find duplicate code in multiple languages like Java, Ruby, Python, and so on.
```

**Comment by ybiquitous** [comment]
Thank you. 😄

**Pull request merged**